packages/core/test/Form_test.js

```diff
@@ -1,10 +1,9 @@
 import { expect } from "chai";
 import sinon from "sinon";
-import React from "react";
+import React, { createRef } from "react";
 import { renderIntoDocument, Simulate } from "react-dom/test-utils";
 import { render, findDOMNode } from "react-dom";
 import { Portal } from "react-portal";
-import { createRef } from "create-react-ref";
```

Comment:
is this a breaking change? are we changing the minimum version of react we support now?

Reply:
if so, we should mention this in the changelog

Reply:
and update package.json accordingly

Reply:
I don't believe this is a breaking change... This is a test and we are on a version of react that has `createRef` already provided.

Reply:
Although you may be correct about the minimum version of react in the core package.json likely needing to be bumped

Reply:
So this means that it's a breaking change as the minimum version of react required to use rjsf is now react 16.3 (as opposed to 16.0)